BUILDING.md

```diff
@@ -106,7 +106,8 @@ You need to install the following
 - Xcode
 - [Doxygen](#doxygen) (only if generating documentation)
 
-For the load tests applications you need to install the [Vulkan SDK](#vulkan-sdk).
+For the load tests applications you need to install the [Vulkan SDK](#vulkan-sdk), and create an environment
```

Comment:
This should say something like

"For the load tests applications you need to install the [Vulkan SDK](#vulkan-sdk) and run the included `setup-env.sh` or otherwise create an environment variable `VULKAN_SDK` pointing to the `macOS` directory within the directory where you installed the SDK."